config/switches.rst

```diff
@@ -204,7 +211,7 @@ debug:
 ~~~~~~
 Single value, type: ``boolean`` (``true``/``false``). Default: ``false``
 
-Set this to true to get additional debug output.
+Set this to true to get additional debug output. You need to set this flag to see event you have defined for this switch in mpf monitor.
```

Comment:
Only in case you are not consuming the event ;-).

Reply:
that is a very good hint, didn't see that with my dummy learning setups for now. Just saw it somewhere in the event reference. I will update my changes in the next days accordingly that others don't miss it either. Thanks for pointing out.